### ISSUE TITLE
Drop item from outside calendar at any point of time

### ISSUE THREAD

**Issue by napatwongchr** [namespace-ee/react-calendar-timeline#709]
**The problem**
I have tested beta version on rowRenderer and I cannot drop item from the outside of the calendar at any point of time

**Solution**
It would be nice if the library provides some functionality to drop items from the outside of the calendar at any point of time.

**Comment by napatwongchr** [comment]
I came up with the solution to do this and I want some feedback because I am not sure that is a decent way to do it so.

The problem is I want to drop the items from outside of the calendar.
1. I get the clientOffset, x coordinate from drop monitor, which is a function from react-dnd.
2. I have used timeFor(e) the function to calculate the dropping point of time that has been placed in HelpersContext, to use in rowRenderer, which contains drop area.
3. So drop function from react-dnd could get the dropping point of time, now I could calculate start_time and end_time to update item in gantt.

Thank you.

**Comment by Ilaiwi** [comment]
@napatwongchr yep this sounds good! The community would appreciate it if you could provide a codesandbox for your solution.

**Comment by janakact** [comment]
@napatwongchr Can't we implement this with a custom renders and react dnd?

**Comment by napatwongchr** [comment]
@janakact Yes it could, you could check it out on the example on 0.27.0-beta release. But from the  example you could only drop on specific slots that you have defined. You have to modify it to make it drop on anywhere on timeline. So, that I came up with solution above.

**Comment by Ruzen1** [comment]
@napatwongchr Bro, Can you please share a working example?

**Comment by developer-ocansey** [comment]
Closing as part of issue triage. This issue was created before 2023 and has had no recent activity. If you're still experiencing this on v0.30+, please open a new issue with a minimal reproduction.
